transport/tlscommon/tls_config_test.go

```diff
@@ -684,12 +684,12 @@ func startTestServer(t *testing.T, serverAddr string, serverCerts []tls.Certific
 	return *serverURL
 }
 
-func getFingerprint(cert *x509.Certificate) string {
+func GetFingerprint(cert *x509.Certificate) string {
```

Comment:
`GetCertFingerprint` now that these are public. `tlscommon.GetCertFingerprint` is a bit more specific of the use case.

Also, a doc comment is welcome as well.

Reply:
+1